Remove obsolete frameworks and runtime identifiers from the project file

smy-huddly opened the pull request:
This serves two purposes:
* Get rid of obsolete stuff from the project
* The runtime identifier "win" is not valid (afaik) and breaks publishing in the camera service

thomas-huddly reviewed (commented):
I think we should keep our changes to this repo minimal. Can we keep `netstandard2.0;net46;`?

smy-huddly committed:
adds net8.0 target

smy-huddly committed:
The runtime identifer 'win' is making our .net6 and .net8 projects fail
to build and publish properly. I've hidden it behind a conditonal to
avoid it. Only tested with .net6 and .net8 so I have no idea how this
change affects building with other frameworks
